python/semantic_kernel/connectors/ai/chat_completion_client_base.py

```diff
@@ -303,8 +306,18 @@ async def get_streaming_chat_message_contents(
                     ],
                 )
 
+                # Merge and yield the function results, regardless of the termination status
+                # Include the ai_model_id so we can later add two streaming messages together
+                # Some settings may not have an ai_model_id, so we need to check for it
+                ai_model_id = self._get_ai_model_id(settings)
+                function_result_messages = merge_streaming_function_results(
+                    messages=chat_history.messages[-len(results) :],
+                    ai_model_id=ai_model_id,  # type: ignore
+                )
+                if self._yield_function_result_messages(function_result_messages):
+                    yield function_result_messages
```

Comment:
I like it, just wondering if this will break folks, if one has logic on what to do with FCC and TextContent and then suddenly you get this as well.. maybe we should add a flag on the class/call that is False by default and that decides whether to send or not.

Reply:
I was thinking about if this is breaking, but the fact that it’s additive, had me thinking it would be okay. My thought was folks would be checking specific item types, so if wanting to handle this, they add a new case.

Reply:
This is probably breaking but the impact is small and it's something we have to fix.